src/en/getting-started-jaas.md

```diff
@@ -0,0 +1,127 @@
+Title: Getting started with Juju as a Service
+
+# Getting started with Juju as a Service
+
+Juju as a Service (JAAS) is the fastest and easiest way to quickly get started
+modeling and deploying your cloud-based applications. When you use JAAS,
+Canonical manages the Juju infrastructure. This frees you to concentrate on
+your software and solutions. With JAAS you can deploy, configure, and operate
+your applications on AWS, GCE, and Azure.
+
+## Create an Ubuntu single sign on account
+
+JAAS uses [Ubuntu single sign on][ubuntuSSO] (SSO) for identity management. If you don’t
+already have an Ubuntu SSO account, create one. You will provide a username
+when registering which is the username that will be used by JAAS.
+
+## Obtain cloud credentials
+
+In order for JAAS to deploy your workload on the cloud of your choice, you must
+enter credentials for each cloud you would like to use.
+
+We recommend that you use each public cloud's identity and access management
+(IAM) tool to generate a new set of credentials exclusively for use with JAAS.
+See [Cloud credentials][credentials] for more information. 
+
+## Use JAAS from the web GUI
```

Comment:
Never mind, I found the request to move the CLI stuff to another page, making my parallel comment obsolete. I can change this.